src/components/StaticMetadata/__stubs__/metadataList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { DescriptionList } from '@amsterdam/bmi-component-library';
+import { Tag } from '@amsterdam/asc-ui';
+
+export const metadataList: DescriptionList = [
```

Comment:
I'm confused about this not throwing an error :thinking: 
I believe the appropriate type would be `ComponentProps<typeof DescriptionList>['list']`